distributor-api-v1/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 18.06.2020
+
+* Removed `AlwaysIncluded` field from [Product](operations.md#product) ([37](https://github.com/MewsSystems/gitbook-distributor-guide/pull/37/files)). This flag does not have any meaning anymore. The API is returning its value always equal to `false`. In order to pre-select a product during a booking, you can use `IncludedByDefault` flag.
```

Comment:
I would remove ` This flag does not have any meaning anymore. The API is returning its value always equal to `false`.` IMHO it says nothing to a developer.